packages/volto/news/6487.bugfix

```diff
@@ -0,0 +1 @@
+Fixed Body class depending on sections @giuliaghisini
```

Comment:
```suggestion
Replace _all_ spaces with `-` in `BodyClass` classes, instead of with `-` or `` depending on the content type or section. @giuliaghisini
```